pkgs/applications/science/logic/ls4/default.nix

```diff
@@ -0,0 +1,38 @@
+{ zlib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "ls4-${version}";
+
+  version = "1.0";
+
+  buildInputs = [ zlib ];
+
+  buildPhase = ''
+  cd core
+  # These object files were committed into the git repo, perhaps accidentally
+  rm -f aiger.o
+  rm -f aiger.o_32
+  gcc -g -O3 -c aiger.c
+  make
+  '';
+
+  installPhase = ''
+  mkdir -p $out/bin
+  cp ls4 $out/bin
+  '';
+
+  src = fetchFromGitHub {
+    owner = "quickbeam123";
+    repo = "ls4";
+    rev = "0293895817ede193277b321a38226abdfdf75bef";
```

Comment:
Please either use ``v${version}`` as revision to match your ``version`` or set ``version`` to ``unstable-2019-07-31`` to match ``0293895817ede193277b321a38226abdfdf75bef``.

Reply:
`unstable-*` is a better choice. I noticed that I included four commits not in version 1.0 with that rev. Thanks for pointing this out!